packages/format-library/src/link/index.native.js

```diff
@@ -86,13 +85,21 @@ export const link = {
 					let startIndex = value.start;
 					let endIndex = value.end;
 
-					while ( find( value.formats[ startIndex ], startFormat ) ) {
+					while (
+						value.formats[ startIndex ].find(
```

Comment:
This change causes a crash. It appears `value.formats[ startIndex ]` sometimes returns `undefined`, which `find` must've gracefully handled previously. 

```suggestion
						value.formats[ startIndex ]?.find(
```

Reply:
Thanks for this! Addressed in 6e7ec5fe30b2db3aa06859ae115fda878a6f8cea.